docs/userGuide.md

```diff
@@ -102,47 +102,21 @@ repository, it does not have to be lightweightm2m-iotagent, although we will use
 git remote add lightweightm2m-iotagent https://github.com/telefonicaid/lightweightm2m-iotagent.git
 ```
 
-Before starting contributing, remember to synchronize the `develop` branch in your forked repository with the `develop`
```

Comment:
I'd suggest to recover this text (using `master` intead of `develop`). It doesn't hurt and may be useful for contributors.

Reply:
Fixed with 41edfcdf6b4da736051e4894ce69f5622376da9a